plugins/carbon_black_defense/requirements.txt

```diff
@@ -1,3 +1,4 @@
 # List third-party dependencies here, separated by newlines. 
 # All dependencies must be version-pinned, eg. requests==1.2.0
 # See: https://pip.pypa.io/en/stable/user_guide/#requirements-files
+# insightconnect-plugin-runtime==4.2.0
```

Comment:
My mistake during debugging was telling you to put this in- you correctly commented it out but this should be removed.